gamebook/scene.py

```diff
@@ -25,9 +28,28 @@ def show_options(self):
         for option in self.options:
             title = option.show_title()
             user_inputs = option.show_user_inputs()
-            nextscene = option.show_next_scene()
+            nextscene = option.show_next_scenes()
             
             choice_list += f"{title} , {user_inputs} , {nextscene}\n"
         
         return choice_list
 
+    def next_scene_draw(self, option_to_evaluate):
+        
+        if len(self.options) == 0:
+            raise ValueError("options is empty")
+        
+        for opt in self.options:
```

Comment:
I would make the logic of getting the correct option a private function (something like `_get_option_by_user_input`)